### ISSUE TITLE
Are we allowed to create a conda env and use that python to install?

### ISSUE THREAD

**Issue by zhuoqun-chen** [NVIDIA-Omniverse/OmniIsaacGymEnvs#78]
In the `README.md`, we are instructed to install by specifying the `PYTHON_PATH` pointing to `python.sh`, and then use that to install via:

```bash
pip install -e .
```

Can we create an empty conda env then install this repo using the new isolated `python` and yet somehow still being able to access the `Isaac Sim` libraries?

Thank you!

**Comment by Pipe-Runner** [comment]
Yes absolutely. That is precisely what I have done, and it works perfectly fine. Feel free to ask for specifics if you need them.
